docker-compose.yml

```diff
@@ -9,17 +9,17 @@ services:
     volumes:
       - pgdata:/var/lib/postgresql/data
 
-  pgweb:
-    image: sosedoff/pgweb
-    environment:
-        - DATABASE_URL=postgres://postgres:postgres@db:5432?sslmode=disable
```

Comment:
did you mean to remove the pgweb image?

Reply:
Yes.  Unless there is a lot of demand for this read-only web ui, we might as well strike it from the `docker-compose.yml`.

Reply:
I have always wondered if anyone uses this, because I usually log into the pgsql image and use psql to check db related stuff.

Reply:
I actually used it! :P